Gemfile

```diff
@@ -1,6 +1,14 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
-# Specify your gem's dependencies in wor-soaparty.gemspec
 gemspec
+
```

Comment:
Correct me if I'm wrong, but isn't declaring the dependencies of other gems in the gemspec enough? Isn't this redundant?